canonical/ubuntu-advantage-client
# proxy confirm changes (SC-153)

## Proposed Commit Message


> proxy: prompt before overwriting existing values
> 
> This applies to any time we're configuring a proxy. We are now checking
> to see if there is already a value, and if it is different from our new
> value, we prompt the user to ask them if we should overwrite.

## Test Steps


- manually set an apt proxy
- try to set the apt proxy via ua, by adding to uaclient.conf and then enabling an apt based service
- verify you are prompted
- test both answers to prompt work as expected

repeat steps for snap and livepatch proxies

## Desired commit type::


## Checklist:
